src/index.php

```diff
@@ -14,25 +15,46 @@
 use Cronbeat\Controllers\LoginController;
 use Cronbeat\Controllers\SetupController;
 use Cronbeat\Database;
+use Cronbeat\Logger;
 use Cronbeat\UrlHelper;
 
+// Initialize logger with INFO level
+Logger::setMinLevel(Logger::INFO);
```

Comment:
@jetbrains-junie remove the comment. also, try loading min logger level via ENV, fallback to INFO if not set.

docker compose should use debug by default.